tests/test_flake8_trio.py

```diff
@@ -341,9 +360,9 @@ def print_first_diff(errors: Sequence[Error], expected: Sequence[Error]):
     for err, exp in zip(errors, expected):
         if err == exp:
             continue
-        if not first_error_line or err.line == first_error_line[0]:
+        if not first_error_line or err.line == first_error_line[0].line:
             first_error_line.append(err)
-        if not first_expected_line or exp.line == first_expected_line[0]:
+        if not first_expected_line or exp.line == first_expected_line[0].line:
```

Comment:
have never worked as intended since it was written I'm pretty sure :sweat_smile: